templates/definition/charger/tesla-ble.yaml

```diff
@@ -0,0 +1,63 @@
+template: tesla-ble
+products:
+  - description:
+      generic: Tesla BLE
+group: generic
+requirements:
+  description:
+    de: Open Source Tesla BLE HTTP Proxy https://github.com/wimaha/TeslaBleHttpProxy
+    en: Open Source Tesla BLE HTTP Proxy https://github.com/wimaha/TeslaBleHttpProxy
+params:
+  - preset: vehicle-common
+  - name: vin
+    required: true
+    example: W...
+    help:
+      de: Erforderlich für BLE-Verbindung
+      en: Required for BLE connection
+  - name: url
+    required: true
+    example: http://192.168.178.27
+    help:
+      de: URL des Tesla BLE HTTP Proxy
+      en: URL of the Tesla BLE HTTP Proxy
+  - name: port
+    example: 8080
+    default: 8080
+    help:
+      de: Port des Tesla BLE HTTP Proxy
+      en: Port of the Tesla BLE HTTP Proxy
+render: |
+  type: custom
+  power:
```

Comment:
@andig I noticed that chargers can take a `power` getter so I'm trying this since `TeslaBleHttpProxy` provides voltage and amps. However, it works fine without this (I assume evcc just estimates this but I haven't found that code yet). Curious if you think it is worth keeping?

Reply:
Dunno. Worth keeping it its better than the estimate.